pvlib/iotools/sodapro.py

```diff
@@ -44,7 +44,7 @@
 def get_cams(latitude, longitude, start, end, email, identifier='mcclear',
              altitude=None, time_step='1h', time_ref='UT', verbose=False,
              integrated=False, label=None, map_variables=True,
-             server='www.soda-is.com', timeout=30):
+             server='api.soda-solardata.com', timeout=30):
```

Comment:
The default value mentioned in the docstring needs to be updated as well

Reply:
@kandersolar I created a new constant and referred to that instead. ``:const:`pvlib.iotools.sodapro.URL``